README.md

```diff
@@ -277,7 +321,7 @@ What follows is a deeper dive into both of these content types.
 
 For any kind of repeating content, this is the basic process:
 
-1. In the vars file for the section you're in (e.g., `blog/_vars-blog.html`),
+1. In the vars file for the section you're in (e.g., `newsroom/_vars-newsroom.html`),
```

Comment:
Is there anything in here that can be removed? --> https://github.com/cfpb/cfgov-refresh/tree/flapjack/cfgov/jinja2/v1/newsroom

I just updated to refer to a file that still exists, would `events/_vars-events.html` be better?